apps/condo/domains/common/utils/crypto.js

```diff
@@ -0,0 +1,12 @@
+const crypto = require('crypto')
+
+/**
+ * Converts value to MD5 hash
+ * @param value
+ * @returns {string}
+ */
+const md5 = (value) => crypto.createHash('md5').update(value).digest('hex')
```

Comment:
it seems that no one will find such utilities except the author. The folder with utilities will turn into a garbage dump. 
It may be easier to add an inline function directly to the file where this function is used. (optional)

Reply:
it's used in several places over the project (code and tests) and will be used in more places soon. so it's better to have one source of truth here.